fixed the url styling for webhooks to now use pure html instead of ma…

svelle opened the pull request:
right now webhook url styling is broken, see:
![image](https://github.com/user-attachments/assets/fb9a9c9b-f0e5-4445-857b-dc8e4efaab77)


I've changed the url to just use regular old html and it now looks like this:

![image](https://github.com/user-attachments/assets/9cec8fa8-518e-4953-aae3-b9d096898d66)
![image](https://github.com/user-attachments/assets/33a9101b-66c3-426e-be60-7a9612362c18)
![image](https://github.com/user-attachments/assets/716c61aa-d886-4794-b610-cabfac0a1c3f)

```release-note
Fixed inconsistent styling in new integration pages
```

hmhealey commented on webapp/channels/src/components/integrations/confirm_integration/confirm_integration.tsx:
```diff
@@ -356,19 +356,19 @@ const ConfirmIntegration = ({team, location, commands, oauthApps, incomingHooks,
                     <FormattedMessage
                         id='add_outgoing_oauth_connection.token_url'
-                        defaultMessage='<b>Token URL</b>: `{url}`'
+                        defaultMessage='<b>Token URL</b>: {url}'
```
You'll also need to update `i18n/en.json` for these changes to the `defaultMessage`s. The current i18n tooling unfortunately doesn't sync changes to the `defaultMessage` with that file

svelle replied:
I checked all the i18n files and I couldn't find any with the backticks in it.. strange

hmhealey commented on webapp/channels/src/components/integrations/confirm_integration/confirm_integration.tsx:
```diff
@@ -135,7 +135,7 @@ const ConfirmIntegration = ({team, location, commands, oauthApps, incomingHooks,
                     defaultMessage='<b>URL</b>: {url}'
                     values={{
-                        url: '`' + incomingHookToken + '`',
+                        url: <span><code style={{display: 'inline'}}>{incomingHookToken}</code></span>,
```
Are you sure that these need the outer `span` element or the `display: inline` on the `code` tag? Playing around with this locally, I think that this would be sufficient
```suggestion
                        url: <code>{incomingHookToken}</code>,
```

svelle replied:
You're correct. I was going off of the <pre><code> example you posted and didn't actually check if just the code tag sufficed 😅 my bad.

hmhealey replied:
I'm trying to remember where I needed `pre` tags around the `code` tags since I swore they were needed. Maybe it was for code blocks in Markdown specifically. Sorry for the confusion though

svelle commented:
@hmhealey I think I addressed all your review comments. updated all the i18n strings that I could find and also updated more token and url displays.

![2025-04-24_16-50_1](https://github.com/user-attachments/assets/b2f7b7f3-6e23-4adf-9ffc-924f3bf951a4)
![2025-04-24_16-50](https://github.com/user-attachments/assets/83a73be7-8a6f-4f80-adf9-ee4907101bf6)
![2025-04-24_16-49_2](https://github.com/user-attachments/assets/973eb59a-010c-427e-9a6e-556a438a04e2)

should we also change the way we display these?
![2025-04-24_16-49_1](https://github.com/user-attachments/assets/fa420fdc-3401-4242-8bc1-a1ad4049b1d9)
![2025-04-24_16-49](https://github.com/user-attachments/assets/a90a41a1-bf2d-4882-b988-719c85d76391)

hmhealey commented:
@svelle I think those failing test cases aren't related to your changes, so let me know if you'd like me to bypass them to merge this in

hmhealey commented:
Sorry, I guess I could've been more helpful than to just ask that and then go away for a week. 😅 

In the future, you can use the E2E/Overridden label lets us skip the E2E checks. Obviously, we shouldn't overuse it, but it's there for when existing tests appear to be genuinely flaking
